packages/shared/src/regex.ts

```diff
@@ -2,6 +2,7 @@ export const emailRegEx = /^\S+@\S+\.\S+$/;
 export const phoneRegEx = /^\d+$/;
 export const usernameRegEx = /^[A-Z_a-z]\w*$/;
 export const passwordRegEx = /^.{6,}$/;
-export const redirectUriRegEx = /^https?:\/\//;
+export const redirectUriRegEx = /^https?:\/\/.*/;
+export const redirectNativeRegEx = /^[a-z][\d_a-z]*(\.[\d_a-z]+)+[\d_a-z]:\/\/[a-z]*$/;
```

Comment:
How about `uriRegEx` and `customSchemeRegEx`?

Reply:
redirectUriRegEx used elsewhere as well, do update that as well? or just `customSchemeRegEx` this works?

Reply:
Just because redirectUriRegEx can be re-used in elsewhere as well, we need to give it a more general name. Otherwise, it should only be used to validate the redirect URIs.

Reply:
Well, decided not to change the name